tc/core/polyhedral/schedule_transforms.cc

```diff
@@ -101,24 +101,44 @@ isl::union_map partialSchedule(
 }
 
 namespace {
+/*
+ * If "domain" is any filter, then intersect "domain" with that filter.
+ */
+isl::union_set anyFilter(const ScheduleTree* node, isl::union_set domain) {
+  if (auto filterElem = node->elemAsBase<ScheduleTreeElemFilter>()) {
+    return domain.intersect(filterElem->filter_);
+  }
+  return domain;
+}
+
+/*
+ * If "domain" is a mapping filter, then intersect "domain" with that filter.
+ */
+isl::union_set mappingFilter(const ScheduleTree* node, isl::union_set domain) {
```

Comment:
Same as above for the naming, otherwise there is a risk of confusing this with the matcher function.